src/unix/linux-core.c

```diff
@@ -151,7 +151,7 @@ int uv__io_check_fd(uv_loop_t* loop, int fd) {
   struct uv__epoll_event e;
   int rc;
 
-  e.events = POLLIN;
+  e.events = POLLIN | UV__POLLPRI;
```

Comment:
same as the 2 cases above, I'll nor mention it anymore for the rest of the platforms